src/System.Management.Automation/engine/Modules/SwitchProcessCommand.cs

```diff
@@ -99,15 +110,22 @@ protected override void EndProcessing()
         /// The arguments to send through to the executable.
         /// Array must have its final element be null.
         /// </param>
+        /// <param name="env">
+        /// The environment variables to send through to the executable in the form of "key=value".
+        /// Array must have its final element be null.
         /// <returns>
         /// An exit code if exec failed, but if successful the calling process will be overwritten.
         /// </returns>
         [DllImport("libc",
-            EntryPoint = "execv",
+            EntryPoint = "execve",
             CallingConvention = CallingConvention.Cdecl,
             CharSet = CharSet.Ansi,
             SetLastError = true)]
-        private static extern int Exec(string path, string?[] args);
+        private static extern int Exec(string path, string?[] args, string?[] env);
+
+        // leverage .NET runtime's native library which abstracts the need to handle different OS and architectures for termios api
+        [DllImport("libSystem.Native", EntryPoint = "SystemNative_ConfigureTerminalForChildProcess")]
```

Comment:
Is this PR intended to pull in both `Switch-Process` fixes?  This DllImport is here from the other one, but isn't used